app/jobs/regular/emit_web_hook_event.rb

```diff
@@ -25,7 +25,7 @@ def execute(args)
           !web_hook.category_ids.include?(args[:category_id]))
 
         raise Discourse::InvalidParameters.new(:payload) unless args[:payload].present?
-        args[:payload] = JSON.parse(args[:payload])
+        args[:payload] = JSON.parse(args[:payload]) unless args[:retry_count].present?
```

Comment:
Instead of doing this, we should just duplicate a copy of the args at the start and use it when retrying